### ISSUE TITLE
MotorStorm Pacific Rift Problem with the multiplayer / no server issue

### ISSUE THREAD

**Issue by MilanRL** [RPCS3/rpcs3#13126]
The game crashes when you try start a race with more than 9 players in the lobby. The lobby can get up until 12 players. This issue only happens to RPCS3 user and not to PS3 user. The devs of the server already checked the problem and said their server haven't got any problems which should be true because like I said this only happens on RPCS3.

**Comment by Megamouse** [comment]
missing log file 🤦

**Comment by redZONErare** [comment]
[mspronline10players.log.gz](https://github.com/RPCS3/rpcs3/files/10326627/mspronline10players.log.gz)

Hope this helps.

In my experience, the game didn't crash. However when 10 or more players are in the lobby, the game fails to start the match for all players playing on RPCS3. You cannot leave the game during this period due to the match "starting", so in turn you have to close the game. This can be annoying to deal with considering the max player count in a single custom lobby is 12 players, which cannot be limited.

If you're wondering, the custom server we've connected to is the PS Rewired server.

**Comment by RipleyTom** [comment]
Looked at the log, there is no rpcs3 specific error in there. I see it calls a
```
·! 0:30:40.631218 {PPU[0x1000003] Thread (PS Button Handler) [HLE:0x01050174, LR:0x0037679c]} cellSysutil: open_msg_dialog(is_blocking=0, type=0x90, msgString=“An error occurred while communicating with the server.

Would you like to try again?”, callback=*0xe821c8, userData=*0x371a0c04, extParam=*0x0)
```

Don't you get this error dialog?

**Comment by redZONErare** [comment]
> Looked at the log, there is no rpcs3 specific error in there. I see it calls a
> 
> ```
> ·! 0:30:40.631218 {PPU[0x1000003] Thread (PS Button Handler) [HLE:0x01050174, LR:0x0037679c]} cellSysutil: open_msg_dialog(is_blocking=0, type=0x90, msgString=“An error occurred while communicating with the server.
> 
> Would you like to try again?”, callback=*0xe821c8, userData=*0x371a0c04, extParam=*0x0)
> ```
> 
> Don't you get this error dialog?

Yes, it does this periodically while in the menu. You just decline when the error pops up and the server continues to work as normal. This happens for everyone (including PS3 users) so I don't think it is related.

**Comment by Danielet89** [comment]
But is it a problem regarding the emulator or from how it was reverse enginereed?

**Comment by Megamouse** [comment]
Is this still an issue?

**Comment by digant73** [comment]
@MilanRL is that still an issue?

**Comment by MilanRL** [comment]
The issue has been fixed and there are no problems for MotorStorm Pacific Rifts multiplayer at the moment

**Comment by digant73** [comment]
> The issue has been fixed and there are no problems for MotorStorm Pacific Rifts multiplayer at the moment

good, so please close this issue

**Comment by digant73** [comment]
@main developers, please close this report confirmed as fixed
